cmds/replay.c

```diff
@@ -816,6 +818,14 @@ static int print_graph_rstack(struct uftrace_data *handle,
 		}
 	}
 
+	if (opts->srcline) {
+		loc = task_find_loc_addr(sessions, task, rstack->time, rstack->addr);
+		if (loc) {
+			str_loc = xmalloc(strlen(loc->file->name) + 11); // 11: length of integer max value + NULL
```

Comment:
Please put the comment above the code (in a separate line).  I also prefer C-style comments (/* ... */) and 80-character length should be considered (while it's a hard limit)..

```c
/* 11 = length of integer max value + NULL */
str_loc = xmalloc(...);
...
```

Reply:
ok.